examples/platforms/raspbian/default

```diff
@@ -31,6 +31,7 @@
 NAT64=1
 DNS64=0
 DHCPV6_PD=0
```

Comment:
This DHCPV6_PD option controls [this script](https://github.com/openthread/ot-br-posix/blob/3e9be206df0860a7a63b7ad82181f95f8ca2fbc7/script/_dhcpv6_pd#L299).

I'm not sure the use case of it. Probably we should deprecate that script (so we can reuse the DHCPV6_PD option in the setup script for specifying the build option OTBR_DHCP6_PD) or merge our implementation into this script. 

@jwhui I wonder if you have some context about that script?

Reply:
This was the PD solution that was originally contributed in 2017. I am OK with replacing that solution.

Reply:
SGTM to keep both in this PR.